.github/workflows/pull_request.yml

```diff
@@ -5,16 +5,40 @@ name: Pull request tests.
 jobs:
   test:
     name: Pull request tests
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-20.04
     steps:
       - name: Checkout code
-        uses: actions/checkout@master
+        uses: actions/checkout@v3
+        with:
+          path: snapshot-streaming
+
+      - name: Checkout tessellation
+        uses: actions/checkout@v3
+        with:
+          path: tessellation
+          repository: Constellation-Labs/tessellation
+          ref: v1.9.1
+          token: ${{ secrets.GITHUB_TOKEN }}
 
-      - name: Set up JDK 1.8
-        uses: actions/setup-java@v1
+      - name: Set up Java
+        uses: actions/setup-java@v3
         with:
-          java-version: 1.8
+          distribution: temurin
+          java-version: 11
+          cache: sbt
+
+      - name: Check sbt version
```

Comment:
I committed this step by accident. I will remove it along with code-review fixes.

Reply:
Fixed